src/java.base/share/classes/jdk/internal/foreign/abi/BindingSpecializer.java

```diff
@@ -150,7 +150,7 @@ static MethodHandle specialize(MethodHandle leafHandle, CallingSequence callingS
 
         new BindingSpecializer(mv, callerMethodType, callingSequence, abi, leafHandle.type()).specialize();
 
-        mv.visitMaxs(0, 0);
+        mv.visitMaxs(-1, -1);
```

Comment:
visitMaxs arguments are ignored with COMPUTE_FRAMES flag, right? any specific reason for -1, -1 instead of 0, 0?

Reply:
I think you are right. I got confused initially as I thought I've seen another place in this class where -1 was used, and I was trying to experiment making ASM spit the bytecode (and not crash) so that I could look at the actual bytecode :-)

This can be reverted